docs/pkglists/CentOS_7.x86_64

```diff
@@ -1,10 +1,8 @@
 debug/x86_64/bareos-debuginfo
```

Comment:
This file should not be part of this commit. I guess, it will disappear with a rebase from master.